src/components/PortfolioCard.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <li 
+    class="card"
+  >
+    <div class="usa-card__container">
+      <header class="usa-card__header">
+        <h2 class="usa-card__heading">
+          {{ viz.title }}
+        </h2>
+      </header>
+      <div class="usa-card__media">
+        <div class="usa-card__img">
+          <img
+            src="https://designsystem.digital.gov/img/introducing-uswds-2-0/built-to-grow--alt.jpg"
+            :alt="viz.title" 
+          >
+        </div>
+      </div>
+      <div class="usa-card__body">
+        <p>
+          {{ viz.description }}
+        </p>
+        <p>
+          Published {{ viz.released }}
```

Comment:
Some of these dates I made up. Would be nice to have the initial sort be based on the release date though